### ISSUE TITLE
Editor test results logging special characters instead of text

### ISSUE THREAD

**Issue by mstellaluna** [freeCodeCamp/learn#53]
Challenge Name
https://learn.freecodecamp.org/responsive-web-design/basic-html-and-html5/link-to-external-pages-with-anchor-elements

Browser Information
Browser Name, Version:
Firefox Quantum 60.0.1 (64-bit)
Chrome: Version 66.0.3359.117 (64-bit)
Operating System: Windows 10 Pro 64-bit
Mobile, Desktop, or Tablet: Dell Latitude Laptop (15")

Issue Description
The test results in the editor are logging special characters rather than the text those characters represent

![image](https://user-images.githubusercontent.com/22984739/40249926-9c61b524-5aa2-11e8-9cd4-45c34ce080f0.png)

**Comment by Bouncey** [comment]
The message need to be in unescaped going into the output console.

**Comment by rishi-amura** [comment]
@Bouncey I would like to work on it, please let me know if it's not been assigned to anyone.

I guess we can use this article for fix it: https://paulschreiber.com/blog/2008/09/20/javascript-how-to-unescape-html-entities/

**Comment by rishiip** [comment]
@Bouncey I would like to work on it, please let me know if it's not been assigned to anyone.

I guess we can use this article for fix it: https://paulschreiber.com/blog/2008/09/20/javascript-how-to-unescape-html-entities/

**Comment by Bouncey** [comment]
It is yours @rishiip :+1:

**Comment by alvinkl** [comment]
@rishiip you can use this function
```javascript
const unescapeString = str => {
  const el = document.createElement('div');
  return str.replace(/\&[#0-9a-z]+;/gi, enc => {
    el.innerHTML = enc;
    return el.innerText;
  });
};

export default unescapeString;
```

Good luck! 👍

**Comment by rishiip** [comment]
@alvinkl Sure 🎉

**Comment by Bouncey** [comment]
@alvinkl @rishiip, we do not need to create any elements for this. It is just string manipulation

**Comment by Bouncey** [comment]
Oh, now I get it. I'll be quiet for a moment

**Comment by rishiip** [comment]
Hi @Bouncey @alvinkl As I am new to React. I am figuring out how method works in react app. I already figured out the file in which I needs to do modification. I'll let you know, if I am not able to fix it. So that you can assign it to someone else. :blush:

**Comment by tchaffee** [comment]
@Bouncey This was fixed with my PR #111 and can be closed.
